cmd/containerd-shim-runhcs-v1/options/runhcs.proto

```diff
@@ -98,6 +98,9 @@ message Options {
 	// The typical example is if Containerd has restarted but is expected to come back online. A 0 for this field is interpreted as an infinite
 	// timeout.
 	int32 io_retry_timeout_in_sec = 17;
+	
+	// global_container_annotations specifies a set of annotations that should be set for every workload container
+	map<string, string> global_container_annotations = 18;
```

Comment:
Maybe a name like `default_container_annotations` would be better, as it doesn't override anything already set by the client. Then if we wanted to in the future we could add a `forced_container_annotations` which does override.

Reply:
sure, makes sense